cmake/QueueNadoConfig.cmake

```diff
@@ -0,0 +1,42 @@
+#.rst:
+# FindQueueNado
+# -------
+#
+# Find libQueueNado
+# QueueNado deals with file access and directory traversal.
+##
+# Result variables
+# ^^^^^^^^^^^^^^^^
+#
+# This module will set the following variables in your project:
+#
+# ``QueueNado_INCLUDE_DIRS``
+#   where to find 
+# Alien.h      CZMQToolkit.h    Headcrab.h         Kraken.h    
+# QueueNadoMacros.h     Rifle.h            Shotgun.h    ZeroMQ.h
+# BoomStick.h  HarpoonBattle.h  IComponentQueue.h  Listener.h  
+# ReceiveDpiMsgLRZMQ.h  SendDpiMsgLRZMQ.h  Skelleton.h
+# Crowbar.h    Harpoon.h        KrakenBattle.h     Notifier.h  
+# ReceivePacketZMQ.h    SendPacketZMQ.h    Vampire.h
+
+
+#
+# ``QueueNado_LIBRARIES``
+#   the libraries to link against to use libQueueNado.
+#   that includes libQueueNado library files.
+#
+# ``QueueNado_FOUND``
+#   If false, do not try to use QueueNado.
+#
+include(FindPackageHandleStandardArgs)
+find_path(QueueNado_INCLUDE_DIR Alien.h CZMQToolkit.h Headcrab.h Kraken.h QueueNadoMacros.h  Rifle.h Shotgun.h ZeroMQ.h BoomStick.h  HarpoonBattle.h IComponentQueue.h Listener.h ReceiveDpiMsgLRZMQ.h  SendDpiMsgLRZMQ.h Skelleton.h Crowbar.h  Harpoon.h KrakenBattle.h Notifier.h ReceivePacketZMQ.h SendPacketZMQ.h Vampire.h
```

Comment:
For this and the other LROSS pull request.   I do not think the `<repo name>Config.cmake` are necessary. Further, they are hard to test also so I believe all of the special cmake config files  should no be installed and in fact  removed
(Murmurhash, FileIO, StopWatch, DeathKnell, Queuenado)